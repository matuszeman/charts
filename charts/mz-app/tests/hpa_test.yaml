suite: hpa
values:
  - ../values-ci.yaml
templates:
  - hpa.yaml
tests:
  - it: should match snapshot with values-ci.yaml
    asserts:
      - matchSnapshot: {}
