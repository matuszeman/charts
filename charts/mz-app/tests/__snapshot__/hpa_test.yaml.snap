should match snapshot with values-ci.yaml:
  1: |
    apiVersion: autoscaling/v2
    kind: HorizontalPodAutoscaler
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: app-name
        app.kubernetes.io/version: 0.1.0
        helm.sh/chart: mz-app-0.1.0
      name: RELEASE-NAME-resource-suffix
    spec:
      maxReplicas: 10
      metrics:
        - resource:
            name: cpu
            target:
              averageUtilization: 80
              type: Utilization
          type: Resource
      minReplicas: 1
      scaleTargetRef:
        apiVersion: apps/v1
        kind: Deployment
        name: RELEASE-NAME-resource-suffix
